src/uipath/_cli/_runtime/_runtime.py

```diff
@@ -36,6 +42,11 @@ async def execute(self) -> Optional[UiPathRuntimeResult]:
         await self.validate()
 
         try:
+            trace.set_tracer_provider(TracerProvider())
+            trace.get_tracer_provider().add_span_processor(  # type: ignore
```

Comment:
why?

Reply:
![image](https://github.com/user-attachments/assets/002112a3-e423-42d9-8760-29d2c57653e5)
I want to set this with a custom processor in uipath-llama-index